packages/nextra/src/compile.ts

```diff
@@ -18,11 +18,8 @@ import { truthy } from './utils'
 import remarkMath from 'remark-math'
 import rehypeKatex from 'rehype-katex'
 
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
 globalThis.__nextra_temp_do_not_use = () => {
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
+  // @ts-expect-error -- ignore error - File is not a module
```

Comment:
to avoid adding `eslint-disable` comment, just use `@ts-expect-error` instead of `@ts-ignore` and describe why the error occurs, this will be less annoying